docs/Developers/Newcomers_Guide.rst

```diff
@@ -23,28 +23,28 @@ following tasks, you will become a developer at coala:
   - merge a ``difficulty/low`` Pull Request
   - review at least a ``difficulty/low`` or higher Pull Request
 
-When you ran coala on a project, please fill our
-`usability survey <http://coala.io/usability>`_. Once you got your first Pull
-Request merged successfully, fill in our
-`survey form <http://coala.io/newform>`_. With that you can help us making your
+Once you've run coala on a project, please fill out our
+`usability survey <http://coala.io/usability>`_. And once you've got your first Pull
+Request merged successfully, fill out our
+`survey form <http://coala.io/newform>`_. By doing so, you can help us make your
 experience better!
 
-Once you have achieved all these, just ask for being promoted on the chat and
-provide links to your reviews and merged Pull Requests. Then, you will be able
-to name yourself a coala developer!
+Once you have achieved all these, just ask to be promoted on the chat
+(see step 1 below) and provide links to your reviews and merged Pull Requests.
+Then you'll be able to call yourself a coala developer!
 
 .. note::
 
-    **Do not only fix a newcomer issue!** Supervising newcomers is really a lot
+    **Don't just fix a newcomer issue!** Supervising newcomers is really a lot
```

Comment:
The `really` makes it redundant because `a lot` is enough to express a big quantity